appinventor/appengine/build.xml

```diff
@@ -633,6 +637,7 @@
       <arg value="-extra"/>
       <arg value="${build.extra.dir}"/>
       <arg line="${client.flags} ${client.module}"/>
+      <arg line="-generateJsInteropExports"/>
```

Comment:
Does this new flag have any extra side effect in any part of the code besides the Mock Extensions?

Reply:
It is needed for the GWT's JSInterop feature. I believe it won't have any side effects elsewhere.